parcels/particleset/particlesetnodes.py

```diff
@@ -1220,7 +1233,7 @@ def execute(self, pyfunc=AdvectionRK4, pyfunc_inter=None, endtime=None, runtime=
         if verbose_progress is None:
             walltime_start = time_module.time()
         if verbose_progress:
-            pbar = self._create_progressbar_(_starttime, endtime)
+            pbar = self.__create_progressbar(_starttime, endtime)
```

Comment:
may I ask why you did rename it ?
I terms of class inheritance in Python, class member functions- and variables written as `_<name>_` are in _protected_ space, meaning: they are callable to visible just to this class and its derived classes. Members of the signature `__<name__` (such as `__init__`) are in _private_ space, meaning, they are just visible and callable by this very class itself. This is how proper access protection works.

Now, in this case, creating the progress bar arguably should be only callable from the ParticleSet and inherited classes (i.e. specialized particle sets), but certainly not from outside this class. That is why I put it in _private_ space. In other words: that naming signature is not arbitrary, but has a purpose in the context of access protection. Unless there is a reasonable argument against it, I tend to reject this renaming.

Reply:
I copied it from https://github.com/OceanParcels/parcels/blob/202ef087ba8608d4950a096569a6992cf278d022/parcels/particleset/baseparticleset.py#L107-L116. In fact, it may even be cleaner to directly inherit from there? Feel free to adapt the code; I just needed a quick fix in order to testdrive the PR

Reply:
No, so I changed that because it was actually done very dirty there, and I did it properly.

Can you perhaps tell me what the problem was for you so we can fix the problem rather than introduce the error source again ? I'd be happy to know what problem was caused from the renaming and the access protection..

Reply:
The problem was that there was no `_create_progressbar` defined in particlesetnodes.py, so as soon as simulations were longer than a few seconds (and the progressbar was created), it raised an error

Reply:
I'll check that again - sounds like a weird error cause that should make the tests fail too, which it doesn't. Are you sure you properly installed the branch ? Again, conversely I'll check that up again.

Reply:
Ah, I see the error - I'll fix it right away. Thanks for your explanation. Really weird I didn't catch the issue earlier.

Reply:
fixed version  ready for pull.